api/src/api/services/phases/handlers/squid-router-phase-handler.ts

```diff
@@ -83,13 +82,12 @@ export class SquidRouterPhaseHandler extends BasePhaseHandler {
         state: {
           ...state.state,
           squidRouterApproveHash: approveHash,
-          squidRouterSwapHash: swapHash,
+          squidrouterSwapHash: swapHash,
```

Comment:
We are getting inconsistent here, let's make it `squidRouter` everywhere and not `squidrouter`.